subsys/net/lib/lwm2m/ipso_voltage_sensor.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2021 Laird Connectivity
```

Comment:
2023. And is the copyright correct here?

Reply:
I took the ipso_current_sensor.c file and modified it to produce the ipso_voltage_sensor.c. Since there is already 2021 copyright of Laird in the ipso_current_sensor,c, should I put the copyright as follow:
Copyright (c) 2021-2022 Laird Connectivity
Copyright (c) 2023 Hydro-Quebec

Reply:
In that case, keep the old one as is, and add your below.